examples/ex011_advection_supg.jl

```diff
@@ -76,6 +77,22 @@ function advection_supg_symbol(θ)
     return sort(imag.(eigvals(-M \ A)))
 end
 
+# transformation matrix
+function transformation_matrix()
+    R = computewavenumbersymbol()
+    return sort(imag.(eigvals(R)))
+end
+
 eigenvalues = hcat(advection_supg_symbol.(θ)...)'
+plot(θ / π, eigenvalues ./ π, linewidth = 3)  # Dispersion
+plot!(
+    identity,
+    xlabel = "θ/π",
+    ylabel = "Eigenvalues",
+    label = "exact",
+    legend = :none,
+    color = :black,
+    title = "P=$P, collocate=$collocate, τ=$τ",
+)
```

Comment:
Manual plotting shouldn't be part of these examples - only the Jupyter examples

Reply:
Yes, I am aware of that, it is only for tentative experimental purposes, it will be eventually removed.